client/disputes/index.tsx

```diff
@@ -205,9 +205,20 @@ export const DisputesList = (): JSX.Element => {
 	const totalRows = disputesSummary.count || 0;
 
 	const rows = disputes.map( ( dispute ) => {
+		const onClickDisputeRow = (
+			e: React.MouseEvent< HTMLAnchorElement >
+		) => {
+			// Use client-side routing to avoid page refresh.
+			e.preventDefault();
+			wcpayTracks.recordEvent( 'wcpay_disputes_row_action_click' );
+			const history = getHistory();
+			history.push( getDetailsURL( dispute.charge_id, 'transactions' ) );
+		};
+
 		const clickable = ( children: React.ReactNode ): JSX.Element => (
 			<ClickableCell
 				href={ getDetailsURL( dispute.charge_id, 'transactions' ) }
+				onClick={ onClickDisputeRow }
```

Comment:
💭 The event name and description point towards this event being specific to the action button rather than the row.

```
This event measures when the action button on a dispute row is clicked.
```

Should the user action of clicking the row be recorded as separate event? i.e. `wcpay_disputes_row_click`. This would be more consistent with deposits: `wcpay_deposits_row_click`.

Or do you think it's OK for us to apply this event name loosely, since it shares the same user intent (viewing the details of a dispute)?

Reply:
> Or do you think it's OK for us to apply this event name loosely, since it shares the same user intent (viewing the details of a dispute)?

I think the button and the row represent the same user action, so am intentionally proposing same event for all. I don't think we want to micro-optimise our UI like that (yet!), if we want to understand merchant behaviour when viewing and actioning disputes, we want to know when the click through to detail view. 

Good point about the event docs, I should update that as a follow up. 

> Should the user action of clicking the row be recorded as separate event? i.e. wcpay_disputes_row_click. This would be more consistent with deposits: wcpay_deposits_row_click.

That's a good comparison. In that table we have a hyperlinked date, and we don't track that separately from clicking the row.

If we were curious to optimise or understand micro details like this, we might add these events … but IMO we need coarse events covering the main actions merchants can take. E.g. view list, click to see detail, etc.